pod.go

```diff
@@ -283,6 +283,10 @@ type Resources struct {
 
 	// Memory is the amount of available memory in MiB.
 	Memory uint
+
+	// Bridges is the list of pci bridges attached to the pod
+	// pci bridges can be used to hot plug devices
+	Bridges []*Bridge
```

Comment:
This should be an hypervisor resource IMO.

Reply:
I think a bridge is also a resource, same as memory and CPU